apps/studio/components/interfaces/Realtime/Policies.tsx

```diff
@@ -0,0 +1,72 @@
+import { PostgresPolicy } from '@supabase/postgres-meta'
+import { useState } from 'react'
+
+import { Policies } from 'components/interfaces/Auth/Policies'
+import { AIPolicyEditorPanel } from 'components/interfaces/Auth/Policies/AIPolicyEditorPanel'
+import { useProjectContext } from 'components/layouts/ProjectLayout/ProjectContext'
+import AlertError from 'components/ui/AlertError'
+import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
+import { useTablesQuery } from 'data/tables/tables-query'
+
+export const RealtimePolicies = () => {
+  const { project } = useProjectContext()
+
+  const [policyEditorShown, showPolicyEditor] = useState(false)
+  const [selectedPolicyToEdit, setSelectedPolicyToEdit] = useState<PostgresPolicy | undefined>(
+    undefined
+  )
```

Comment:
if im not wrong this should be okay
```suggestion
  const [selectedPolicyToEdit, setSelectedPolicyToEdit] = useState<PostgresPolicy>()
```

Reply:
Fixed.